src/utils/pgwire/src/pg_protocol.rs

```diff
@@ -131,7 +128,11 @@ where
             Err(e) => {
                 tracing::error!("Error: {}", e);
                 match e {
-                    PsqlError::CancelError(_) | PsqlError::SslError(_) | PsqlError::IoError(_) => {}
+                    PsqlError::SslError(io_err) | PsqlError::IoError(io_err) => {
```

Comment:
Better use '@' here. 
https://stackoverflow.com/a/49906536

Reply:
How to use '@' here..